src/controllers/sheets.ts

```diff
@@ -156,9 +157,17 @@ const addApplicants = async (
                 userId: googleId,
             });
             let new_sheets: Array<Schema.Types.ObjectId> = [];
+            let level = 'Beginner';
+            let status = 'Pending Applications';
+            let screeningGrade = undefined;
+            let interviewGrade = undefined;
 
             if (iApplicant != null) {
                 new_sheets = iApplicant.sheets;
+                level = iApplicant.level;
+                status = iApplicant.status;
+                screeningGrade = iApplicant.screeningGrade;
+                interviewGrade = iApplicant.interviewGrade;
```

Comment:
One nit here is that I wouldn't name the queried applicant `iApplicant`, "i" or "I" is better reserved only for where you are referencing the actual interface.

Reply:
Sure, I'll change that. I'm not sure why it was done that way.